k8s/namespaces/nfdiv/nfdiv-cron-remind-applicants-apply-for-final-order/aat-00.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: nfdiv-cron-remind-applicants-apply-for-final-order
+spec:
+  releaseName: nfdiv-cron-remind-applicants-apply-for-final-order
+  values:
```

Comment:
Seems like your cronjob doesn't override env vars per environment using the environment variable?
https://github.com/hmcts/nfdiv-cron/blob/master/nfdiv-cron/values.yaml#L20-L34

Reply:
There was a reason but I haven't got the foggiest as to what it was: https://github.com/hmcts/nfdiv-cron/commit/2e43dc956d2230033c8a821cfd5affd0f64eb6a6